modules/imgproc/test/test_intelligent_scissors.cpp

```diff
@@ -158,14 +191,14 @@ TEST(Imgproc_IntelligentScissorsMB, rect)
 
     Point target_point(100, 30);
     std::vector<Point> pts;
-    tool.getContour(target_point, pts);
+    getAndCheckContour(tool, target_point, pts);
 
     tool.applyImage(getTestImage2());
 
     tool.buildMap(source_point);
 
     std::vector<Point> pts2;
-    tool.getContour(target_point, pts2, true/*backward*/);
+    getAndCheckContour(tool, target_point, pts2, true/*backward*/);
 
     EXPECT_EQ(pts.size(), pts2.size());
```

Comment:
`getAndCheckContour` checks contour size too. The following size checks are not required here and bellow.